Orange/widgets/widget.py

```diff
@@ -478,6 +478,12 @@ def copy_to_clipboard(self):
 
     def storeControlAreaVisibility(self, visible):
         self.controlAreaVisible = visible
+        
+        """Set minimum width to 1 (overrides minimumSizeHint) when control area 
+        is not visible to allow the main area to shrink further. Reset the minimum 
+        width with a 0 if control area is visible."""
+        self.controlArea.setMinimumWidth(int(not visible))
```

Comment:
Set the minimum width on `self.left_side` (or better `self.__spliter.widget(0)`). This does not work for e.g *Data Table* or any other widget that specifies the `buttons_area_orientation`)